src/context/alert.tsx

```diff
@@ -1,10 +1,40 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
-import React, { createContext, FunctionComponent, useState } from "react";
+import React, {
+  createContext,
+  FunctionComponent,
+  useState,
+  useCallback
+} from "react";
 import {
   AlertModal,
   AlertModalProps
 } from "../components/AlertModal/AlertModal";
 
+export const ERROR_MESSAGE = {
```

Comment:
I think all these errors have a period at the end too!

Reply:
oh! i was referencing the previous error messages that didnt have a period behind! should we add that?

Reply:
added periods after asking Eida!